hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1647,10 +1650,141 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 		}
 	}
 
+	// Reconcile etcd-backup cronJob
+	{
+		serviceAccount := manifests.EtcdBackupServiceAccount(hcp.Namespace)
+		if _, err = createOrUpdate(ctx, r.Client, serviceAccount, NoopReconcile); err != nil {
+			return ctrl.Result{}, fmt.Errorf("failed to reconcile etcd-backup cronJob service account: %w", err)
+		}
+
+		cronJob := manifests.EtcdBackupCronJob(hcp.Namespace)
+		if _, err = createOrUpdate(ctx, r.Client, cronJob, func() error {
+			return r.reconcileEtcdBackupCronJob(cronJob, serviceAccount, hcluster)
+		}); err != nil {
+			return ctrl.Result{}, fmt.Errorf("failed to reconcile etcd-backup cronJob: %w", err)
+		}
+	}
+
 	log.Info("successfully reconciled")
 	return ctrl.Result{}, nil
 }
 
+func (r *HostedClusterReconciler) reconcileEtcdBackupCronJob(cronJob *batchv1.CronJob, serviceAccount *corev1.ServiceAccount, hcluster *hyperv1.HostedCluster) error {
+	cronJob.Spec = batchv1.CronJobSpec{
+		Schedule: "0 */12 * * *", // TODO
```

Comment:
thinking we might want to have a determinist way to tweak this so we can e2e CI test it.

Reply:
A agree that this should be configurable, but default should be hourly rather than every 12 hours.